src/services/fileManager.ts

```diff
@@ -134,21 +134,29 @@ export async function saveAndIndexFilesFromRdpcData(
 
 export async function recalculateFileState(file: File) {
   // If the file is not already released, lets update its embargo stage
-  const updates: any = {};
+  const updates: { embargoStage?: EmbargoStage; releaseState?: FileReleaseState } = {};
   const embargoStage = getEmbargoStage(file);
   if (file.releaseState === FileReleaseState.PUBLIC) {
     if (embargoStage !== EmbargoStage.PUBLIC) {
       // A currently public file has been calculated as needing to be restricted
       // Record the calculated embargoStage but the file remains correctly listed as releaseState = PUBLIC
-      updates.embargoStage = embargoStage;
+      logger.debug(
+        'recalculateFileState()',
+        file.fileId,
+        'PUBLIC file embargo calculated to be',
+        embargoStage,
+        'Setting release state to',
+        FileReleaseState.QUEUE_TO_RESTRICTED,
+      );
+      updates.releaseState = FileReleaseState.QUEUE_TO_RESTRICTED;
```

Comment:
Here we control stopping the file from being set to RESTRICTED immediately, instead tracking it in the queue for future restricted.